map.config.json

```diff
@@ -0,0 +1,1073 @@
+{
```

Comment:
good idea to have a config, but it's way too complex though, you should have the minimal config to make it work and understand how the map is built.